.github/workflows/test.yml

```diff
@@ -1,23 +1,73 @@
-on: [push, pull_request]
+on: push
```

Comment:
Why not on PR?

Reply:
It's runs them an all pushes, so do we need pr as well?

Reply:
:thinking: maybe only on pull_request is better instead of on pushes